src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormDetails.tsx

```diff
@@ -0,0 +1,316 @@
+import {
+  Box,
+  Clickable,
+  Column,
+  Flex,
+  GridColumns,
+  Input,
+  LabeledInput,
+  Radio,
+  RadioGroup,
+  Select,
+  Text,
+  useToasts,
+} from "@artsy/palette"
+import { ArtworkSidebarClassificationsModalQueryRenderer } from "Apps/Artwork/Components/ArtworkSidebarClassificationsModal"
+import { ArtistAutoComplete } from "Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtistAutocomplete"
+import { useFormikContext } from "formik"
+import { useState } from "react"
+import { ArtworkModel } from "../Utils/artworkModel"
+import { categoryOptions } from "../Utils/categoryOptions"
+import { rarityOptions } from "../Utils/rarityOptions"
+import { ProvenanceModal } from "./ProvenanceModal"
+
+export const MyCollectionArtworkFormDetails: React.FC = () => {
+  const { sendToast } = useToasts()
+
+  const [isRarityModalOpen, setIsRarityModalOpen] = useState(false)
+  const [isProvenanceModalOpen, setIsProvenanceModalOpen] = useState(false)
+
+  const { values, handleChange, setFieldValue, handleBlur } = useFormikContext<
+    ArtworkModel
+  >()
+
+  const isLimitedEdition = values.attributionClass === "LIMITED_EDITION"
+
+  const handleAutosuggestError = (isError: boolean) => {
+    if (isError) {
+      sendToast({
+        variant: "error",
+        message: "An error occurred",
+        description: "Please contact support@artsymail.com",
+      })
+
+      return
+    }
+
+    setFieldValue("artistName", "")
+    setFieldValue("artistId", "")
+  }
+
+  return (
+    <>
+      <ArtworkSidebarClassificationsModalQueryRenderer
+        onClose={() => setIsRarityModalOpen(false)}
+        show={isRarityModalOpen}
+        showDisclaimer={false}
+      />
+      <ProvenanceModal
+        onClose={() => setIsProvenanceModalOpen(false)}
+        show={isProvenanceModalOpen}
+      />
+
+      <GridColumns>
+        <Column span={6}>
+          <ArtistAutoComplete
+            onError={() => handleAutosuggestError(true)}
+            required
+          />
+        </Column>
+        <Column span={6} mt={[30, 0]}>
```

Comment:
Note that `30` isn't in our design sys [spacing scale](https://palette.artsy.net/atoms/spacing/#all-spacing-values) or [theme](https://github.com/artsy/palette/blob/main/packages/palette-tokens/src/themes/v3.tsx#L117-L131) and under no circumstances should we ever step outside of the theme (here and elsewhere) 🙏 

If the designs have spacing values that are outside of that, that's a design bug and devs are responsible for catching it and fixing (devs are the last line of defense, so to speak). Just round up or down to the closest theme value and don't even worry about starting a discussion (other than to maybe gently remind designer that 30 is a nonexistent value per the design sys spec).

Reply:
cc @MrSltun